docs/source/dev_guide/releasing-e3sm-diags.rst

```diff
@@ -161,120 +161,29 @@ This will have the ``conda/meta.yaml`` we edited in the first section.
-
-    ::
-
-        ~/anaconda2/bin/anaconda upload ...
-        ~/anaconda3/bin/anaconda upload ...
-
-If you're having permission issues uploading a package to either of these channels,
-contact either Jill Zhang (zhang40@llnl.gov) or Rob Jacob (jacob@anl.gov) for permission.
-
-
-6. Check the https://anaconda.org/e3sm/e3sm_diags page to view the newly updated package.
-
-
-7. Notify the maintainers of the unified E3SM environment about the new release on the
+3. Notify the maintainers of the unified E3SM environment about the new release on the
 `E3SM Confluence site <https://acme-climate.atlassian.net/wiki/spaces/WORKFLOW/pages/129732419/E3SM+Unified+Anaconda+Environment>`_.
+Be sure to only update the ``e3sm_diags`` version number in the correct version(s) of
+the E3SM Unified environment. This is almost certainly one of the versions listed under
+“Next versions”. If you are uncertain of which to update, leave a comment on the page
+asking.
 
 
 
 Creating a New Version of the Documentation
 -------------------------------------------
 
-The main documentation page includes the most up-to-date information. This means it may contain information on
-features not included in a previous release. The main documentation page, :ref:`index-label`, does link to
-the documentation for previous releases.
-
-After you have released a new version of E3SM Diags, create a new version of the documentation with the following steps:
-
-    ::
-
-        # cd into e3sm_diags directory
-        git fetch <fork-origin> gh-pages
-        git checkout -b <branch-name> <fork-origin>/gh-pages
-        conda activate sphinx
-        make html
-        # Copy the latest docs. They now won't be updated regularly.
-        cp -r docs/html docs/html-v2-2-0 # Replace v2-2-0 with the new version number.
-        # Copy the latest source. This will be useful if previous versions of the docs have to be updated to fix mistakes.
-        cp -r source source-v2-2-0 # Replace v2-2-0 with the version number.
-
-Then, edit ``source/index.rst``, adding the link to the new version of the docs to the list of previous versions.
-This will be of the following form:
-
-    ::
-
-        `v2.2.0 <https://e3sm-project.github.io/e3sm_diags/docs/html-v2-2-0/index.html>`_
-
-Just replace ``v2.2.0`` and ``v2-2-0`` with the new version number.
-
-Run ``make html`` to see how your changes are rendered. Once you are satisfied, commit and push them back to
-the repository: ::
-
-   $ git add .
-   $ git commit
-   $ git push <fork-origin> <branch-name>
-
-Then, create a pull request from ``<your-fork>/e3sm_diags/branch-name`` to ``E3SM-Project/e3sm_diags/gh-pages``.
+The CI/CD workflow will automatically build the most up-to-date version of the docs.
+See :ref:`Continuous Integration / Continuous Delivery (CI/CD) <ci-cd>`.
+If you've released the software on Anaconda (see previous section), then the workflow
+will automatically create a new version of the docs too.
```

Comment:
In CI/CD, the doc building and Anaconda steps are independent of each other.

This does remind me that we also need to trigger doc builds on releases, not just PR merges. A new issue will be created to address this.

Reply:
We might not need this section since it is already automated.

Reply:
Ok, so you're saying `tagging a release on GitHub -> {Anaconda Release & docs release}` rather than `tagging a release on GitHub -> Anaconda Release -> docs release`?

Reply:
I think we should still keep this section, so people reading the docs are aware that new documentation will be published.

Reply:
> Ok, so you're saying `tagging a release on GitHub -> {Anaconda Release & docs release}` rather than `tagging a release on GitHub -> Anaconda Release -> docs release`?

Correct, both steps are performed in parallel